traffic_ops/testing/api/v4/coordinates_test.go

```diff
@@ -212,8 +212,12 @@ func SortTestCoordinatesDesc(t *testing.T) {
 	for start, end := 0, len(respDesc)-1; start < end; start, end = start+1, end-1 {
 		respDesc[start], respDesc[end] = respDesc[end], respDesc[start]
 	}
-	if respDesc[0].Name != respAsc[0].Name {
-		t.Errorf("Coordinates responses are not equal after reversal: %s - %s", respDesc[0].Name, respAsc[0].Name)
+	if len(respDesc[0].Name) > 0 && len(respAsc[0].Name) > 0 {
+		if respDesc[0].Name != respAsc[0].Name {
+			t.Errorf("Coordinates responses are not equal after reversal: %s - %s", respDesc[0].Name, respAsc[0].Name)
+		}
+	} else {
+		t.Errorf("Coordinates name shouldn't be empty while sorting the response")
```

Comment:
This change seems unrelated to this PR

Reply:
Just added null checks, and PR related to theis functionality has closed. Do you want me to include these changes in the respective PR?

Reply:
yes, that would help keep this one cleaner

Reply:
Okay.

Reply:
removed this changes